test/plugins/features/movies/controller.test.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const Controller = require('../../../../lib/plugins/features/movies/controller');
+
+describe('movie controller', () => {
+
+  describe('create', () => {
+
+    it('creates a movie', async () => {
+      const payload = { title: 'WALL-E' };
+
+      const movie = await Controller.create(payload);
+
+      expect(movie.get('title')).to.eql(payload.title);
+
```

Comment:
nit: remove newline

Reply:
Sure. Should I remove it from the notion doc also?